packages/cspell/src/app/util/glob.ts

```diff
@@ -34,7 +36,7 @@ export async function globP(pattern: string | string[], options?: GlobOptions):
     const onlyFiles = options?.nodir;
     const dot = options?.dot;
     const patterns = typeof pattern === 'string' ? [pattern] : pattern;
-    const useOptions: FastGlobOptions = clean({
+    const useOptions: TinyGlobbyOptions = clean({
```

Comment:
you seem to not have added the `expandDirectories` option, is that intentional?

Reply:
thanks for the reminder! I set it on the other PR I sent today, but forgot it one this one